backend/app/Entities/User.php

```diff
@@ -40,6 +41,11 @@ public function sendPasswordResetNotification($token)
         $this->notify(new MailResetPasswordNotification($token));
     }
 
+    public function sendSuccessRegistrationNotification(User $user, string $token)
+    {
+        $this->notify(new MailSuccessRegistrationNotification($user, $token));
```

Comment:
such things better do in action. It looks like user sends email itself, but email should be send by some action. So better do $user->notify(new MailSuccessRegistrationNotification($user, $token))